### DISCUSSION TITLE
Google fonts integration in V 3.x.x

### DISCUSSION THREAD

**Discussion by riyajath-ahamed** [chakra-ui/chakra-ui#9189] [Q&A] [answered]
Hi everyone!
I'm working on a project using Chakra UI and would like to integrate a Google Font (e.g., Roboto) into my theme. However, I'm a bit unsure about the best way to achieve this. Could someone guide me or point me to an example?

@segunadebayo

**Comment by segunadebayo** [comment, marked as answer]
Should be live in the docs shortly, but you can check here:

https://github.com/chakra-ui/chakra-ui/commit/a713cfdeb88f740e7c186a3c1af9035ba5124498
